http-watch.sh

```diff
@@ -97,31 +89,23 @@ do
 		then
 			LOG_DATE
 			STATE=false
-			echo "$log_running_date - Site still offline..." >> $log_file
+			echo "$log_running_date - Website is still offline..." >> $log_file
 		fi
 		
 		while [ $STATE == "false" ]
 		do	
 			LOG_DATE
 			$ACTION
-			echo "$log_running_date - Site still offline... Re-running command..." >> $log_file
+			echo "$log_running_date - Website is still offline... Re-running action command..." >> $log_file
 			PAUSE
 			TESTURL
 			if [ $STATUSCODE == "200" ]
 			then
 				LOG_DATE
 				STATE=true
-				echo "$log_running_date - Site is back online!" >> $log_file
+				echo "$log_running_date - Website is back online!" >> $log_file
 			fi
 		done
 	fi
 	NUMBEROFTRIES=($SAVENUM) # Resets the NUMBEROFTRIES counter backs to its original number
 done
-
-if [ $LOGGING = "1" ]
-then
-	echo "Log file saved at - $log_file"
-elif [ $LOGGING = "0" ]
-then
-	rm ./log_tmp.txt
```

Comment:
I had a look through the code and found that the log_tmp.txt file was replaced by log_$DATE.txt and should no longer be in the latest commit. Because of this, I was unable to find the creation point for 'log_tmp.txt' specifically.

Reply:
Sorry bud, I think that may have been my mistake, I think it looks good now, I can't see the creation of the tmp file.